content/hi/docs/reference/glossary/job.md

```diff
@@ -0,0 +1,20 @@
+---
+title: जॉब (Job)
+id: job
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/job/
+short_description: >
+  एक परिमित या बैच कार्य जो पूरा होने तक चलता है।
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक परिमित या बैच कार्य जो पूरा होने तक चलता है।
+
+<!--more--> 
+
+एक या अधिक {{<glossary_tooltip term_id="pod" >}} ऑब्जेक्ट बनाता है और सुनिश्चित करता है कि उनमें से एक निर्दिष्ट संख्या सफलतापूर्वक समाप्त हो जाए। जैसे ही पॉड्स सफलतापूर्वक पूरा होता है, जॉब सफल पूर्णता को निगाह रखता है।
```

Comment:
The first part of this sentence sounds incomplete in Hindi. Please could we go beyond the machine translation and actually translate this sentence?

Reply:
sure @divya-mohan0209 ,I will try to improve it again

Reply:
@divya-mohan0209 , I have made some changes. please check and give your suggestion